### ISSUE TITLE
[BUG] wash build appears to ignore key directory setting from wasmcloud.toml

### ISSUE THREAD

**Issue by autodidaddict** [wasmCloud/wash-archive#620]
**Describe the bug**
When I supply the `key_directory` field (and for giggles, I also tried `keys_directory`) in the `[actor]` section, `wash build` will still either reuse or generate a new key in `~/.wash/keys`. It appears to the end user that wash build is ignoring this field in the toml file.

**To Reproduce**
1. Put a key in `./keys/my_actor_module.nk` where `my_actor` corresponds to the project name in `Cargo.toml`
2. Add `key_directory=./.keys` to the `wasmcloud.toml` file
3. Run `wash build`, you will see that it has generated a new key and _not_ reused the one from the `keys` directory.

**Expected behavior**
I expect it to honor the `key_directory` setting

**Screenshots**
This is the shell output, note that it's creating a new key:
```
{"keygen":"true","path":"/Users/kevin/.wash/keys/foo_aggregate_module.nk","status":"No keypair found"}

Actor built and signed and can be found at "/Users/kevin/code/(blah blah)"
```

**Environment**
 - OS: macOS
 - Shell zsh
 - wash Version 0.18.0

**Comment by connorsmith256** [comment]
From what I can tell, `key_directory` is parsed from the actor config, but isn't used anywhere :( 

https://github.com/wasmCloud/wash/blob/67d9b91085ed74c64ef8bc56de48e54e4a760a64/crates/wash-lib/src/parser/mod.rs#L48

The `--keys-directory` option on `wash build` does appear to be used, FWIW

**Comment by connorsmith256** [comment]
Closed by #926
